vendordeps/photonlib.json

```diff
@@ -1,7 +1,7 @@
 {
     "fileName": "photonlib.json",
     "name": "photonlib",
-    "version": "v2024.2.10",
+    "version": "v2024.3.1",
```

Comment:
This is a major update, so we need to reinstall PhotonVision on the pis before we merge this or it will not work.